mcp-agent-server/src/types.ts

```diff
@@ -0,0 +1,62 @@
+import type { JSONSchema7 } from "json-schema";
+
+export interface AgentSpec {
+  /** Optional human-friendly name used in tool description */
+  readonly name?: string;
+  /** Optional human-friendly description used in tool description */
+  readonly description?: string;
+  /**
+   * Optional custom tool name. Must match /[a-zA-Z0-9_]+/.
+   * If omitted, we will generate `run_<slug(name)>`.
+   */
+  readonly toolName?: string;
+  /** JSON Schema for the agent-specific input parameters (first argument). */
+  readonly inputSchema?: JSONSchema7;
+  /** Optional JSON Schema for the agent result payload. */
+  readonly outputSchema?: JSONSchema7;
+  /** Static parameters merged into every run (e.g., agentId). */
+  readonly staticParams?: Record<string, unknown>;
+}
+
+export interface AgentRuntimeOptions {
+  /** Optional set of tools the agent may call; passed through to the agent. */
```

Comment:
@CharlieHelps is there a way to borrow these parameters from `RunAgentInput` and/or `RunAgentParameters` in `@ag-ui/core` ?